packages/atlas/src/components/_video/VideoTileViewer/VideoTileViewer.tsx

```diff
@@ -47,7 +47,7 @@ export const VideoTileViewer: React.FC<VideoTileViewerProps> = ({ id, onClick, d
       videoHref={videoHref}
       channelHref={channelHref}
       onChannelAvatarClick={() => navigate(channelHref)}
-      loadingDetails={loading}
+      loadingDetails={loading || !video}
```

Comment:
Maybe we should use `internalIsLoadingState` that is returned from useVideoTileShared logic then? it seems, that we're doing the same thing twice: 
```tsx
  const internalIsLoadingState = loading || !video
```
You'll need to also update `VideoTilePublisher`

Reply:
I'm not sure if this is good idea, since `loading` and `video` are passed from `VideoTileViewer`
`VideoTilePublisher` already has this change.